internal/vendors/accessor_spec.go

```diff
@@ -0,0 +1,12 @@
+package vendors
+
+type AccessorGetAllPaginationSpec struct {
+	Limit  int
+	Offset int
+	Order  string
+}
+
+type AccessorGetAllPaginationData struct {
+	Vendors      []Vendor
+	TotalEntries int
```

Comment:
I think we need more data on the pagination instead of just the total entries, since we have to serve these data for the UI
- number of total pages (not yet)
- numer of entries in total (not yet)
- the current page number (not yet)

```suggestion
	PaginationMetadata paginationMetadata
```
```
// helper.go
type PaginationMetadata struct {
	TotalPages         int
	TotalEntries       int
	CurrentPage        int
	// TotalEntries is optional, since we could determine that from `[]Vendor` length for example, but anyways is fine
}
```

I think for now we don't need to implement a wrapper for the accessor, since that might be more complex
just a simple function would suffice IMO